docusaurus.config.ts

```diff
@@ -15,17 +18,17 @@ const config: Config = {
   },
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-site.example.com',
+  url: isDev ? 'http://localhost:3000' : 'https://layer-3.github.io',
```

Comment:
Please ensure this does not affect running it from the developer's local machine